speechToText.py

```diff
@@ -2,36 +2,57 @@
 import pyttsx3
 import keyboard
 
+import time
+
 # Initialize the recognizer
 recognizer = sr.Recognizer()
+recognizer.energy_threshold = 300
+
+# text to audio ==>
+def speak_text(text):
+    engine = pyttsx3.init()
+    engine.say(text)
+    engine.runAndWait()
+
+# Function to listen to the microphone with key binding
+def listen_untill_release():
+    audio_data = bytearray()  # Store raw audio data here
 
-# Function to listen to the microphone
-def record_text():
     with sr.Microphone() as source:
-        print("Listening...")
```

Comment:
add listening statement so that user knows that computer is listening

Reply:
Okay, I updated in next commit👍